main-settings/src/main/scala/sbt/Append.scala

```diff
@@ -84,4 +84,31 @@ object Append {
       b
     }
   }
+
+  // un-implicit, package-private (and JVM public) the old way of defining these
+  private[sbt] trait Sequence[A, -B, T] extends Value[A, T] with Values[A, B]
```

Comment:
Cool. I wasn't completely sure if private[sbt] was jvm public. I'm now wondering if I should change a bunch of watch related deprecated apis to package private instead so that we can turn -Xfatal-warnings on in the main project.

Reply:
The upside of deprecated is it can include a message (and a version), while package private just says "isn't visible".

Reply:
Oh, we also might want to also deprecate these even though they're package private so we don't forget to remove them in sbt 2.

Reply:
Done